src/pages/login.js

```diff
@@ -8,8 +8,8 @@ function Login() {
 
   const handleSubmit = async (event) => {
     const data = new FormData(event.target);
-    const mail = data.getElementById("email").value;
-    const pass = data.getElementById("password").value;
+    const mail = data.get("email").value;
```

Comment:
Does this work? If the input has `name="email"` and `name="password"`, then you don't need the `.value` to access the value

Reply:
Ups you're right. But yes it worked even with .value :sweat_smile: